src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import {
+    Typography, 
+    AppBar, 
+    Toolbar,
+    IconButton
+} from '@material-ui/core';
+import { 
+    Menu as MenuIcon, 
+} from '@material-ui/icons';
+
+const Navbar: React.FC = () => {
+    return (
+      <AppBar position="sticky" color="secondary">
```

Comment:
color as "background.dp04".
However it won't compile until on master branch, but what other choice do we have?

Reply:
I can rebase to your branch xD

Reply:
Hmm, not sure about this one - AppBar defined  color props as:
```
  type Color = 'inherit' | 'primary' | 'secondary' | 'default';
```

I am not sure if we can customise it. I will check that though